1-js/05-data-types/02-number/article.md

```diff
@@ -137,15 +137,15 @@ Here's the table to summarize the differences between them:
 |`-1.6`|  `-2`    |   `-1`  |    `-2`  |   `-1`   |
 
 
-These functions cover all possible ways to deal with the decimal part as a whole. But what if we'd like to round the number to `n-th` digit after the point?
+These functions cover all of the possible ways to deal with the decimal part of a number. But what if we'd like to round the number to `n-th` digit after the decimal?
 
 For instance, we have `1.2345` and want to round it to 2 digits, getting only `1.23`.
 
-There are two ways to do so.
+There are two ways to do so:
 
 1. Multiply-and-divide.
 
-    For instance, to round the number to the 2nd digit after the point, we can multiply the number by `100`, call the rounding function and then divide back.
+    For example, to round the number to the 2nd digit after the decimal, we can multiply the number by `100`, call the rounding function, and then divide back.
```

Comment:
Same question: "just decimal", not "decimal point"?

Also, is the comma after "function" really required?

Reply:
see above for the decimal point question. as for the comma, this is a long standing point of contention in English grammar. Google "Oxford comma" for more info. I think the main goal is just to remain consistent. I noticed early on that the articles consistently DO NOT use the Oxford comma, so I avoided added them.

Reply:
Well, I did not know about the Oxford comma, now I do know. So funny different languages have different comma rules for precisely the same syntax constructs.

Reply:
It is difficult, but I agree if it's not present in the other sections as such, then it is OK to remove to keep consistency. I'll retract the comma but add 'it': and then divide **it** back.